configure.ac

```diff
@@ -19,11 +19,11 @@
 ######################################################################
 dnl Process this file with autoconf to produce a configure script.
 
-AC_PREREQ(2.59)
-AC_INIT(s3fs, 1.90)
-AC_CONFIG_HEADER([config.h])
+AC_PREREQ([2.71])
```

Comment:
This change causes CI to fail:

```
configure.ac:22: error: Autoconf version 2.71 or higher is required
```

Do you need the latest version?  Not even Fedora 35 packages 2.71.  I think we could change to 2.69 if necessary which both CentOS 7 and Ubuntu 18.04 offer.

Reply:
Reverted to 2.59 at 478266f .
The warning was:
```
$ ./autogen.sh
--- Make commit hash file -------
--- Finished commit hash file ---
--- Start autotools -------------
configure.ac:24: warning: The macro `AC_CONFIG_HEADER' is obsolete.
configure.ac:24: You should run autoupdate.
../autoconf-2.71/lib/autoconf/status.m4:719: AC_CONFIG_HEADER is expanded from...
configure.ac:24: the top level
configure.ac:26: warning: The macro `AC_CANONICAL_SYSTEM' is obsolete.
configure.ac:26: You should run autoupdate.
../autoconf-2.71/lib/autoconf/general.m4:2081: AC_CANONICAL_SYSTEM is expanded from...
configure.ac:26: the top level
--- Finished autotools ----------
```